packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -22,8 +22,17 @@ const AccountSignup = ({ enableApp, isModalVisible, clients_country, onSignup, r
     const history_value = React.useRef();
     const [pw_input, setPWInput] = React.useState('');
     const [is_password_modal, setIsPasswordModal] = React.useState(false);
-    const disableBtn = (values, errors) =>
-        !values.residence || !!errors.residence || !values.citizenship || !!errors.citizenship;
+    const [is_disclaimer_accepted, setIsDisclaimerAccepted] = React.useState(false);
+
+    const checkResidenceIsBrazil = selected_country =>
+        selected_country && residence_list[indexOfSelection(selected_country)]?.value?.toLowerCase() === 'br';
```

Comment:
@yashim-deriv in the residence_list WS response, there is residence code `br` for brazil. i have used it for the check. hope this is fine.